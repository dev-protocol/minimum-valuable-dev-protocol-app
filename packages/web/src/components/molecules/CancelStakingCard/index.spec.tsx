import React from 'react'
import { render } from '@testing-library/react'
import { CancelStakingCard } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${CancelStakingCard.name}`, () => {
  test('Snapshot', () => {
    const component = render(<CancelStakingCard onClickCancel={() => {}} onClickWithdraw={() => {}} />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
