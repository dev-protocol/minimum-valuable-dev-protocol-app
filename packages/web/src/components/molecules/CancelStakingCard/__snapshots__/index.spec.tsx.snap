// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelStakingCard Snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-card ant-card-bordered"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="ant-row"
        >
          <div
            class="ant-col"
            style="display: flex; align-items: center; flex: 1 1 252px;"
          >
            <span
              class="sc-AxjAm ldzVrk"
            >
              Cancel Staking
            </span>
          </div>
          <div
            class="ant-col"
            style="flex: 1 1 252px;"
          >
            <div
              class="ant-space ant-space-horizontal ant-space-align-center"
            >
              <div
                class="ant-space-item"
                style="margin-right: 16px;"
              >
                <button
                  class="ant-btn ant-btn-primary ant-btn-lg"
                  disabled=""
                  type="button"
                >
                  <span>
                    Cancel
                  </span>
                </button>
              </div>
              <div
                class="ant-space-item"
                style="margin-right: 16px;"
              >
                <button
                  class="ant-btn ant-btn-primary ant-btn-lg"
                  type="button"
                >
                  <span>
                    Withdraw
                  </span>
                </button>
              </div>
              <div
                class="ant-space-item"
              >
                <div />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
